src/content/ru/2021/pwa.md

```diff
@@ -0,0 +1,600 @@
+- `icon`: это свойство появляется на 84,69% десктопных и 86,11% мобильных сайтов. Иконки используются в различных местах: на домашнем экране, в переключателе задач операционной системы и т.д. Это объясняет его широкое распространение.
+- `start_url`: это свойство присутствует на 82,84% десктопных и 84,66% мобильных сайтов. Это ещё одно важное свойство для PWA, поскольку оно указывает, какой URL будет открыт, когда пользователь запускает веб-приложение.
+- `display`: это свойство объявлено на 86,49% десктопных и 87,67% мобильных сайтов. Оно используется для обозначения режима отображения веб-сайта. Если оно не указано, значением по умолчанию является `browser`, который является обычной вкладкой браузера, поэтому большинство PWA объявляют его, чтобы указать, что вместо этого сайт должен быть открыт в режиме `standalone`. Возможность открываться в автономном режиме — одна из вещей, которая помогает создать «нативное» впечатление.
+- `prefer_related_applications`: это свойство присутствует на 6,87% сайтов для десктопов и 7,66% сайтов для мобильных устройств, что кажется низким процентом по сравнению с остальными свойствами в этом списке. Причина в том, что Lighthouse не требует, чтобы оно присутствовало, он только предлагает не устанавливать для него значение `true`.
+
+Далее мы углубимся в свойства, которые позволяют нам определять набор значений. Чтобы понять, какие из них наиболее широко используются.
+
+### Самые популярные размеры иконок в манифестах {top-manifest-icon-sizes}
+
+{{ figure_markup(
+  image="pwa-top-pwa-manifest-icon-sizes.png",
+  caption="Самые популярные размеры иконок в манифестах.",
+  description="Гистограмма показывает самые популярные размеры иконок в манифестах для десктопов и мобильных. `192x192` применяется на 72,58% десктопных PWA-сайтов и 74,20% мобильных, `512x512` — на 70,09% и 70,36% соответственно,`144x144` — на 32,76% и 32,59%, `96x96` — на 23,73% и 23,39%, `48x48` — на 23,10% и 22,50%, `72x72` — на 21,03% и 21,30%, `128x128` — на 16,19% и 16,30%, `152x152` — на 14,11% и 14,52%, `384x384` — на 12,93% и 11,84%, `256x256` — на 11,73% и 11,86%, `16x16` — на 8,10% и 8,32%, `36x36` — на 7,36% и 7,01%, `64x64` — на 6,18% и 6,38%, `32x32` — на 6,05% и 6,37%, и, наконец, `120x120` — на 6,14% and 5,61%.", chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTHHnqNdpRUjoeTfsN9_irK57PvZn_Q2X842RLl-RL4ibWmZFvO-S1x35PjVE3-xUlHFS_Zurd22rOq/pubchart?oid=1930133957&format=interactive",
+  sheets_gid="1450624146",
+  sql_file="top_manifest_icon_sizes.sql"
+  )
+}}
+
+Самыми популярными размерами иконок на сегодняшний день являются 192x192 и 512x512, которые <a hreflang="en" href="https://web.dev/add-manifest/#icons">рекомендуются в Lighthouse</a>. На практике разработчики также предоставляют другие размеры, чтобы они хорошо смотрелись на различных экранах устройств.
+
+### Самые популярные значения свойства display {top-manifest-display-values}
```

Comment:
It's a property name. We write it in our IDEs as-is, not in Russian. I think, Russian web-developers clearly understand `display`, but they will be confused with the translation, because it's not standardized. I can wrap it to `<span lang="en">display</span>`, but it looks like we should do it in every complete translation if it is ok.

Reply:
OK let's leave for now. We've been very inconsistent with the use of spans to wrap (the French team were great at this, but the rest less so). Not a biggie.